### PR TITLE
Fixes so that GeoServer can be packaged for OS X

### PR DESCRIPTION
**Pull request by dmkelsey**

### CODE REVIEW

File: build/mac_installer.sh [hunk 1]
```diff
@@ -27,98 +27,22 @@ fi
...
-     end tell
-   end tell
-' | osascript
-
-# convert to compressed image, delete temp image
-hdiutil convert "${dmg_tmp}" -format UDZO -imagekey zlib-level=9 -o "${dmg_final}"
-if [ -f "${dmg_tmp}" ]; then
-  rm -f "${dmg_tmp}"
-fi
+pushd tmp/geoserver-$tag-mac/ > /dev/null
```
Comment (jodygarnett) [inline build/mac_installer.sh]: It feels like you are killing tones of stuff here?

**Comment by jodygarnett** [comment]
As per the CONTRIBUTING.md file we ask that you have an individual contributors agreement or a corporate contributors agreement to provide changes to OSGeo. In this case you are covered by Boundless.

**Comment by jodygarnett** [comment]
Normally I would ask you to squash this to a single commit (for back porting) - do you want to give it a go?

Since we always use the build scripts from "master" it is not strictly necessary, just neater.

**Comment by jodygarnett** [comment]
Unless @aaime corrects me we do no usually create Jira issues for build issues (or if we do we just mark them closed and do not provided a fixedVersion - preventing them appearing in release notes).

File: build/mac_installer.sh (hunk 1 above)
Comment (dmkelsey) [inline build/mac_installer.sh]: The `./build.sh` script, which lives in the `geoserver-$tag-mac` director duplicates everything I removed..  The difference is that `./build.sh` works correctly.

**Comment by jodygarnett** [comment]
Thanks mutely

**Pull request merged**